extensions/amp-izlesene/amp-izlesene.md

```diff
@@ -19,19 +19,19 @@ limitations under the License.
 <table>
   <tr>
     <td width="40%"><strong>Description</strong></td>
-    <td>Displays a <a href="https://www.izlesene.com/">Izlesene</a> video.</td>
+    <td>Displays an embedded <a href="https://www.izlesene.com/">Izlesene</a> video.</td>
   </tr>
   <tr>
     <td width="40%"><strong>Availability</strong></td>
-    <td>Stable</td>
+    <td>Coming Soon; Available on <a href="https://www.ampproject.org/docs/reference/experimental#opt-into-the-amp-dev-channel">Dev Channel</a></td>

```

Comment:
In a few days, we should revert this back to `stable`. @bpaduch would you please set a reminder to do this?